SU-HSDO/suhumsci
# fix:(STN-167):  dropdown text

# READY FOR REVIEW

## This card is not a priority for sparkbox-sprint-6, please dont review till others are ready and merged.

## Summary

[**JIRA CARD**](https://sparkbox.atlassian.net/browse/STN-167)
Fixes two small issues we found with the chosen js exposed filter dropdown:

- line height needed to be a bit taller
- descenders of text (such as g, y,) were being cut off

(See issue for screenshots and details)

## Steps to Test
1. Run `npm test` in the humsci_basic folder and confirm that all tests and linters pass.
1. Import the horizontal card view: [horizontal-card-config.txt](https://github.com/SU-HSDO/suhumsci/files/4149105/horizontal-card-config.txt)
1. Create a basic page with a view using the horizontal card with list display:
    <img width="276" alt="Screen Shot 2020-02-03 at 12 17 19 PM" src="https://user-images.githubusercontent.com/2486846/73674767-3445c500-467f-11ea-95ab-2d9eabb564db.png">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
